### DISCUSSION TITLE
How to specify a tsconfig file path?

### DISCUSSION THREAD

**Discussion by xcatliu** [vuejs/vitepress#1365] [Q&A]
Hi there, thanks for the awesome work on vitepress!

I tried vitepress in my old project, but it throw an error, after debugged it I found the reason is that vitepress use my project's `tsconfig.json` to build the site.

I read the docs and searched for it, but no answers for this issue.
So how can I specify a tsconfig file path in the vitepress cli?

**Comment by brc-dd** [comment]
Create a tsconfig inside your docs directory? Or if it's not separate then use references to override stuff?
